server.py

```diff
@@ -308,5 +309,31 @@ def _get_comments_at_label_indices(source, label_indices):
 def preferences():
     return render_template("preferences.html")
 
+def amud_is_last(masechet,amud):
```

Comment:
I'd move this into masechtot, as I think it would be relevant beyond just this file